schema.sql

```diff
@@ -12,3 +12,5 @@ CREATE TABLE animals (
     NEUTERED BOOLEAN, 
     WEIGHT_KG DECIMAL
 );
+
+ALTER TABLE animals ADD SPECIES TEXT;
```

Comment:
- [ ] [OPTIONAL] You did well with this query. But I see that used a data type `text` instead of its counter party `varchar(255)`. Below is a brief comparison between them. In SQL, `VARCHAR(255)` and `TEXT` are both data types used to store character string data. However, there are some differences between them:

- Storage size: VARCHAR(255) has a fixed storage size that is equal to the length of the characters it's storing, up to the maximum length of 255 characters. On the other hand, TEXT has a variable storage size and can store up to 2 GB of text.

- Retrieval performance: Retrieving data from a VARCHAR(255) column is generally faster than from a TEXT column because VARCHAR(255) columns are stored in-row, while TEXT columns are stored out-of-row.
- Indexing: VARCHAR(255) columns can be indexed, which can improve lookup performance. TEXT columns can be indexed as well, but with some technical limitations.
- Usage: VARCHAR(255) is commonly used for storing shorter text values such as names, addresses, or emails which are usually shorter than 255 characters. TEXT is used to store large text values like long-form content, descriptions, or comments.

In summary, VARCHAR(255) is ideal for storing relatively short text values while TEXT is better suited for storing large amounts of text data. It is important to choose the appropriate data type based on the nature of the data and its usage requirement as it can affect storage, performance, and retrieval time.